runelite-api/src/main/java/net/runelite/api/widgets/WidgetInfo.java

```diff
@@ -153,6 +153,7 @@
 	BANK_SETTINGS_BUTTON(WidgetID.BANK_GROUP_ID, WidgetID.Bank.SETTINGS_BUTTON),
 	BANK_TUTORIAL_BUTTON(WidgetID.BANK_GROUP_ID, WidgetID.Bank.TUTORIAL_BUTTON),
 
+	GROUP_STORAGE_UI_CONTAINER(WidgetID.GROUP_STORAGE_GROUP_ID, WidgetID.GroupStorage.TITLE),
```

Comment:
Changed my mind slightly on the naming here, just removing the trailing `_CONTAINER`. (I'll commit this via GitHub's UI, no need for you to make any changes.)
```suggestion
	GROUP_STORAGE_UI(WidgetID.GROUP_STORAGE_GROUP_ID, WidgetID.GroupStorage.TITLE),
```